docs/md_files/3_achtergronden_en_uitgangspunten/e_onderliggende_rasterdata/1_onderliggende_rasterdata.md

```diff
@@ -0,0 +1,106 @@
+
+De onverharde infiltratiecapaciteit wordt vermenigvuldigd met drie reductiekaarten: 
+1. Verhardings reductiekaart
+
+    Deze kaart reduceert de onverharde infiltratiecapaciteit ter plaatse van verharding en water. 
+
+2. Erf reductiefactor kaart.
+
+    Deze kaart reduceert de onverharde infiltratiecapaciteit op particuliere erven ten gevolge van partiële verharding. 
+
+3. Riolering reductiefactor kaart 
+
+    Deze kaart geeft een additionele infiltratie reductie in stedelijk gebied om een oppervlakkige afvoercomponent te creëren waarmee het weglaten van riolering wordt gecompenseerd. 
+
+Kaarten 2 en 3 worden afgeleid uit een kaart die de dichtheid van bebouwing aangeeft: de bebouwingsfractie kaart.
+
+### **Rasterkaart Bodemberging**
+De hoeveelheid bodemberging hangt af van het grondgebruik, het bodemprofiel en de ontwateringsdiepte. In de 3Di poldermodellen worden 3 bodemberging rasters aangemaakt, behorend bij 3 verschillende grondwaterstanden (GLG, GHG en het gemiddelde daarvan GGG). Omdat het GGG raster het gemiddelde is van het GHG en GLG raster zijn alleen de GHG en GLG rasters getoond.
+
+Voor gedetailleerde regionale toepassingen bestaan er geen betrouwbare gebiedsdekkende databronnen voor GLG/GHG. Er bestaan wel gebiedsdekkende GLG/GHG kaarten maar die zijn dusdanig onnauwkeurig dat ze voor regionale toepassingen lokaal grote fouten kunnen introduceren. Voor het beheergebied van HHNK bestaan er 2 gebiedsdekkende GLG/GHG producten: 
+1. Alterra GLG/GHG kaart
+
+    De GLG/GHG kaart van Alterra is een rasterkaart van 25 
+    m bij 25 m met de grondwaterstanden in m minus maaiveld. De grondwaterstanden zijn gebaseerd op meetreeksen, de AHN1 en de bodemkaart.
+
+2. De GLG/GHG uit het grondwatermodel van HHNK. 
+
+    Deze (100m bij 100 m) raster kaarten zijn in 2014 gegenereerd met het (MODFLOW/ SEAWAT) grondwatermodel van HHNK en hebben een grove modelschaal (ca. 100 bij 100 m). Deze modelversie betreft een modelvernieuwing van het oorspronkelijke model uit het project Leven met Zout Water uit 2012.
+
+NB: de grondwatertrappen uit de bodemkaart zijn een oudere en minder geavanceerde aanpak op basis van dezelfde brondata als de Alterra kaart en vormt dus geen additionele bron van informatie. 
+
+Naast dit kaartbeeld bestaan er twee methoden waarmee gebiedsdekkend een inschatting gemaakt kan worden van de GLG/GHG standen: 
+1. GHG kaart op basis van ontwateringsnormen:
+
+2. GLG/GHG op basis van drainagebasis.
+
+Tijdens het BWN2-project zijn bergingskaarten gebaseerd op de Alterra grondwaterstanden en bergingscoëfficiënten uit SOBEK. HHNK werkt aan het verbeteren van de input voor de bodembergingskaarten voor toekomstige analyses.
```

Comment:
Dit soort haakjes in tekst zetten dat er aan gewerkt wordt op zich prima, maar hoe zorgen we er dan voor dat we dit ook goed bijwerken als we er daadwerkelijk wat aan gedaan hebben?